src/components/header/template.njk

```diff
@@ -56,38 +55,29 @@
         {% endif %}
       </a>
     </div>
-
-    {% if params.serviceName or params.navigation or params.signOutHref or params.languageToggle %}
+    {% if params.serviceName or params.navigation  %}
     <div class="govuk-header__content">
-      {% if params.serviceName %}
-        <a href="{{ params.serviceUrl }}" class="govuk-header__link govuk-header__link--service-name">
-          {{ params.serviceName }}
-        </a>
-      {% endif %}
-
-      {% if params.navigation %}
-        <button type="button" role="button" class="govuk-header__menu-button js-header-toggle"
-                aria-controls="navigation"
-                aria-label="Show or hide Top Level Navigation">Menu
-        </button>
-        <nav>
-          <ul id="navigation"
-              class="govuk-header__navigation {{ params.navigationClasses if params.navigationClasses }}"
-              aria-label="Top Level Navigation">
-            {% for item in params.navigation %}
-              {% if item.href and item.text %}
-                <li class="govuk-header__navigation-item{{ ' govuk-header__navigation-item--active' if item.active }}">
-                  <a class="govuk-header__link"
-                     href="{{ item.href }}"{% for attribute, value in item.attributes %} {{ attribute }}="{{ value }}"{% endfor %}
-                  >
-                  {{ item.text }}
-                  </a>
-                </li>
-              {% endif %}
-            {% endfor %}
-          </ul>
-        </nav>
-      {% endif %}
+    {% if params.serviceName %}
+    <a href="{{ params.serviceUrl }}" class="govuk-header__link govuk-header__link--service-name">
+      {{ params.serviceName }}
+    </a>
+    {% endif %}
+    {% if params.navigation %}
```

Comment:
Isn't the idea of this change to not show the navigation if an empty list is provided?

Reply:
Yeah, I would have thought the same. So should be:
```suggestion
    {% if params.navigation | length %}
```

Reply:
Hmm, I got this from GDS.  It's a decision between consistency and correctness I guess.  I'm happy to add it.

Reply:
Is this not what the PR is for though? Just asking because I might be misinterpreting the reason for the PR. Should this be upstreamed if this is accepted? I think the navigation might have been changed slightly recently so may not be applicable anymore.